app/controllers/case_contacts_controller.rb

```diff
@@ -5,6 +5,8 @@ class CaseContactsController < ApplicationController
   before_action :set_contact_types, only: %i[new edit update create]
   before_action :require_organization!
   after_action :verify_authorized
+  skip_before_action :authenticate_user!, only: :case_contacts_creation_times_in_last_week
```

Comment:
It would be cleaner to put metrics methods in a different controller

Reply:
The method is now in health_controller.rb moved from case_contacts_controller.rb